core/src/main/java/org/opensearch/sql/expression/DSL.java

```diff
@@ -29,9 +26,10 @@
 import org.opensearch.sql.expression.span.SpanExpression;
 import org.opensearch.sql.expression.window.ranking.RankingWindowFunction;
 
-@RequiredArgsConstructor
```

Comment:
`UtilityClass` is experimental and previously we've ran into some strange issues. I recall there was once it failed the entire javac compile with hundreds of errors because of unused imports.

Reply:
🤦